docusaurus-docs/package.json

```diff
@@ -18,6 +18,7 @@
     "@docusaurus/core": "^3.4.0",
     "@docusaurus/preset-classic": "^3.4.0",
     "@docusaurus/theme-mermaid": "^3.4.0",
+    "plugin-image-zoom": "github:flexanalytics/plugin-image-zoom",
```

Comment:
Why do we use it from GitHub and not from npm?

Reply:
Updated it now